korman/properties/modifiers/water.py

```diff
@@ -446,7 +446,7 @@ class PlasmaWaveGeoState(PlasmaWaveState, PlasmaModifierProperties):
 
     _min_length_default = 4.0
     _max_length_default = 8.0
-    _amplitude_default = 10
+    _amplitude_default = 0.001
```

Comment:
Shouldn't this be `0.1`?

Reply:
Most wavesets I've seen, in Cyan's work, in the old AlcScript, etc, the geostate's amplitude was quite small, usually around 0.0019953...etc. The reason some have been getting "wobbly" waveset geometry (including me) was because, even at 1%/0.01, that was set too high.

Reply:
Ok, I just wasn't expecting to see the default values to change (except to match the calculation changes made in the export routine) based off the PR description.